react/src/Mpool.js

```diff
@@ -13,7 +15,7 @@ export function MpoolPage(props) {
 
 function MpoolContent(props) {
     const [local, setLocal] = useState(true)
-    const {loading, error, data} = useQuery(MpoolQuery, { variables: { local } })
+    const {loading, error, data} = useQuery(MpoolQuery, { variables: { local } , pollInterval: 4000, fetchPolicy: "network-only"})
```

Comment:
Why pool every 4 seconds? This should be at least 15sec. given that block time is 30sec.

Reply:
Actually, this accounts for devnet as well. Ideally, I wanted to use build.BlockDelaySecs but that comes from MpoolQuery so I went with shortest available number. Even with 4 seconds, it should not be too aggressive as we are not polling directly to lotus node.

Reply:
Why do we care about this feature at all on a devnet? There should be no alerts or backlogged messages on a devnet. I'd rather we don't spam Lotus with unnecessary API calls -- all the pages start to add up at some point.

Reply:
I am pushing one more commit to remove all messages page and everything related to that. I will also update the polling time.

This page does not make a call to lotus API directly. It uses cached entries in mpoolManager with Boost.

Reply:
@nonsense I have adjusted the polling time and moving the other changes (Removing all messages) to a new PR.